code/game/objects/structures/statues.dm

```diff
@@ -6,13 +7,20 @@
 	density = TRUE
 	anchored = FALSE
 	max_integrity = 100
+	CanAtmosPass = ATMOS_PASS_DENSITY
 	var/oreAmount = 5
 	var/material_drop_type = /obj/item/stack/sheet/iron
-	CanAtmosPass = ATMOS_PASS_DENSITY
+	/// Beauty component mood modifier
+	var/impressiveness = 15
+	/// Art component subtype added to this statue
+	var/art_type = /datum/element/art
+
+/obj/structure/statue/Initialize(mapload)
+	. = ..()
+	AddElement(art_type, impressiveness)
 
 /obj/structure/statue/attackby(obj/item/W, mob/living/user, params)
 	add_fingerprint(user)
-	user.changeNext_move(CLICK_CD_MELEE)
```

Comment:
Hmm?

Reply:
Random click cooldown is random. Why is it there? Nobody knows...